IRP/src/Catalogs/Partners/Forms/ChoiceForm/Module.bsl

```diff
@@ -67,4 +67,13 @@ EndProcedure
 Procedure SetBooleanListFilter(FilterItems, FieldName, RightValue)
 	CommonFunctionsClientServer.SetFilterItem(FilterItems, FieldName, RightValue, DataCompositionComparisonType.Equal,
 		RightValue = True);
+EndProcedure
+
+&AtClient
+Procedure NotificationProcessing(EventName, Parameter, Source)
+	If EventName = "WritingNew" 
```

Comment:
Rename event name "WritingNew" to "NewPartnerCreated"

Reply:
+